### PR TITLE
Explains Riemann Integration methods

### LINKED ISSUE #30368: Riemann Integration
### Feedback

Hi. Please would you consider adding advice on how to change the "Intergral method" for a Riemann sum integral Helper.?
I have created a few, selecting the default "Trapezoidal" and since learned I probably should have selected "Left".

Also, what does the "SI" mean in the description for "Integration time" option, please?
If I wanted to generate a sum for daily energy consumption from a power sensor, should I select "d"?
That seems to create a helper with unit kWd, but I'm after a kWh total for each day.

Apologies if I have misunderstood how to use this integration/helper and it's just intended to provide a cumulative sensor.

### URL

https://www.home-assistant.io/integrations/integration/

### Version

2023.12.3

### Additional information

Thanks in advance of your kind response.

### PR DESCRIPTION
**Pull request by dgomes**
## Proposed change

Attempts to address user difficulties by providing more information about methods and the functioning of the integration

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/integration.markdown [hunk 1]
```diff
@@ -97,6 +97,24 @@ In case you expect that your source sensor will provide several subsequent value
 
 The unit of `source` together with `unit_prefix` and `unit_time` is used to generate a unit for the integral product (e.g. a source in `W` with prefix `k` and time `h` would result in `kWh`). Note that `unit_prefix` and `unit_time` are _also_ relevant to the Riemann sum calculation. 
 
+## Integration method
+
+Riemann Sum is a approximation of an integral by a finite sum and is therefore intrinsicaly innacurate, nonetheles, depending on the method used, values can be more or less accurate.
+
+Regardless of the method used the integration will be more accurate if the source updates more often. If your source is not updated, neither will the Riemann Sum sensor will, as all this integration does is calculate the next step on the event of a source update.
```
Comment (cadamini) [inline source/_integrations/integration.markdown]:
```suggestion
Regardless of the method used the integration will be more accurate if the source updates more often. If your source is not updated, neither will the Riemann Sum sensor, as all this integration does is calculate the next step in the event of a source update.
```

Comment (dgomes) [reply source/_integrations/integration.markdown]: Thank you!

File: source/_integrations/integration.markdown [hunk 2]
```diff
@@ -97,6 +97,24 @@ In case you expect that your source sensor will provide several subsequent value
 
 The unit of `source` together with `unit_prefix` and `unit_time` is used to generate a unit for the integral product (e.g. a source in `W` with prefix `k` and time `h` would result in `kWh`). Note that `unit_prefix` and `unit_time` are _also_ relevant to the Riemann sum calculation. 
 
+## Integration method
+
+Riemann Sum is a approximation of an integral by a finite sum and is therefore intrinsicaly innacurate, nonetheles, depending on the method used, values can be more or less accurate.
```
Comment (dannytsang) [inline source/_integrations/integration.markdown]:
```suggestion
Riemann Sum is a approximation of an integral by a finite sum and is therefore intrinsically inaccurate, nonetheless, depending on the method used, values can be more or less accurate.
```

File: source/_integrations/integration.markdown [hunk 3]
```diff
@@ -97,6 +97,24 @@ In case you expect that your source sensor will provide several subsequent value
...
+
+Regardless of the method used the integration will be more accurate if the source updates more often. If your source is not updated, neither will the Riemann Sum sensor, as all this integration does is calculate the next step in the event of a source update.
+
+### Trapezoidal
+
+The `trapezoidal` method follows the [Trapezoidal rule](https://en.wikipedia.org/wiki/Trapezoidal_rule). This method is the most accurate of the currently implemented methods, **if** the source updates often, since it better fits the curve of the intrinsic source. 
+
+### Left
+
+The `left` method follows the [Left rule](https://en.wikipedia.org/wiki/Riemann_sum#Left_rule). The method **underestimates** the intrinsic source, but is extremely accurate at estimating rectangular functions which are very stable for long periods of time and change very rapidly (e.g. such as the power function of a resistive load can jump instantly to given value and stay at the same value for hours). If your source keeps its state for long periods of time, this method is preferable to the `trapezoidal`.
```
Comment (dannytsang) [inline source/_integrations/integration.markdown]:
```suggestion
The `left` method follows the [Left rule](https://en.wikipedia.org/wiki/Riemann_sum#Left_rule). The method **underestimates** the intrinsic source, but is extremely accurate at estimating rectangular functions which are very stable for long periods of time and change very rapidly (e.g. such as the power function of a resistive load can jump instantly to a given value and stay at the same value for hours). If your source keeps its state for long periods of time, this method is preferable to the `trapezoidal`.
```

**Review by dannytsang — CHANGES_REQUESTED** [review]

File: source/_integrations/integration.markdown (hunk 2 above)
Comment (dgomes) [reply source/_integrations/integration.markdown]: tks

**Review by dannytsang — APPROVED** [review]

**Comment by dannytsang** [comment]
Thank you @dgomes

**Pull request merged**